app/controllers/api/v1/customers_controller.rb

```diff
@@ -97,6 +97,8 @@ def create_params
             :payment_provider,
             :provider_customer_id,
             :sync,
+            :gocardless_mandate_id,
```

Comment:
Just to make sure I understand well, for gocardless we will have the two identifier `provider_customer_id` and `gocardless_mandate_id`?

Reply:
yep